partner_contact_vat/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+VAT in contact partners
+=======================
+
+This module was written to extend the functionality of contacts management to
+support setting a VAT in contact partners (those partners that are not not
+companies) and allow you to have separate VAT codes for contact and for their
+company.
+
+The company's VAT will still be used for invoiced and other legal documents.
```

Comment:
Odoo by default only allows VAT in parent companies. When a contact belongs to a company and you go to *accounting* page, it shows *Accounting is managed in the parent company*.

If you manage any kind of personal information of your clients (prevention of occupational hazards, training, event assistants, etc.) you probably need their VAT.

Indeed the company VAT will still be used for invoices and such, but for other things you need the person's. For that reason, instead of overwriting the VAT field, I just added a new one that will be hidden for companies.